components/MdContent.vue

```diff
@@ -20,3 +20,32 @@ export default class extends Vue {
   }
 }
 </script>
+
+<style lang="scss" scoped>
+  ul, ol {
```

Comment:
Please, remove the first-level indentation:
```suggestion
ul, ol {
```